docs(headings): fix example headings

jschuler opened the pull request:
- Add headings where it was missing
- Made headings a required prop
- Add variablesRoot prop to all component/layout examples (does not do anything currently in the workspace docs but will in the website)

patternfly-build commented:
Deploy preview for *pf-next* ready!

Built with commit 7fa116db8e2ed55f8af38a92451513707d900d88

https://deploy-preview-1502--pf-next.netlify.com

mcoker commented:
Should "Copy to Clipboard" change to "Clipboard Copy"?

Looks like we still need one for "Tab Content", "Wizard", and "Alignment" (utility).

Do we care about the Demos or Upgrade Examples?

jschuler commented:
@mcoker Thanks, added missing headings. Actually you can again use any text you want in the heading, it was too much to expect the heading to conform to the css variable root name. Instead I added a new variablesRoot prop to all component/layout examples. Don't need them for utilities/demos/upgrades since there are no variables I think.

patternfly-build commented:
:tada: This PR is included in version 1.0.214 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@patternfly/patternfly)
- [GitHub release](https://github.com/patternfly/patternfly-next/releases/tag/v1.0.214)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
